docs/swagger.yaml

```diff
@@ -14,7 +14,7 @@ externalDocs:
   url: https://github.com/andela/dahlia-ah-backend
 
 servers:
-- url: /api/v1
+- url: http://localhost:3000
```

Comment:
I observed you specified port 3000 above if another person is not using the same port number do you think it will still work.